### DISCUSSION TITLE
Support Git based dependencies using clojure.tools.deps

### DISCUSSION THREAD

**Discussion by gaberger** [mikub/titanoboa#23] [Ideas]
Currently doesn't look like git deps are supported in external-dependencies. This would be very helpful in building up reusable workflow functions in the platform

**Comment by mikub** [comment]
This might be useful, following would need to be thought through however:
1. reloading during runtime
2. re-loading git dependencies in clustered setup

These two points are basically the reason we do not have this already since it is not that straightforward.
Currently the choice is between maven repo or prototyped clojure/java anonymous functions with former being preferred for production use the latter for quick prototyping...

If we can't figure out 1. and 2. above then maybe we can enable git just for a single server use just for prototyping/testing...
